src/cli/requireWebpackConfig.js

```diff
@@ -103,7 +103,7 @@ export default function requireWebpackConfig(webpackConfig, required, env, mode)
   config = config.default || config;
 
   if (typeof config === 'function') {
-    config = config(env);
+    config = await Promise.resolve(config(env));
```

Comment:
Why to use `Promise.resolve` here and not just `await`?

Reply:
If `config` is not a `Promise`, this causes an error in the test at line 40 of `test/unit/cli/requireWebpackConfig.test.js` because `config` as a regular `Function` is not thenable. Therefore, using `Promise.resolve` here is to ensure that a `Promise` is always returned.

![image](https://user-images.githubusercontent.com/8129489/69960647-9a6f6980-14cf-11ea-8602-4d8aaebdc291.png)